packages/create-app/src/lib/versions.ts

```diff
@@ -33,92 +33,92 @@ import { version as root } from '../../../../package.json';
 
 import { version as appDefaults } from '../../../app-defaults/package.json';
 import { version as backendDefaults } from '../../../backend-defaults/package.json';
-import { version as catalogClient } from '../../../catalog-client/package.json';
 import { version as catalogModel } from '../../../catalog-model/package.json';
 import { version as cli } from '../../../cli/package.json';
 import { version as config } from '../../../config/package.json';
 import { version as coreAppApi } from '../../../core-app-api/package.json';
 import { version as coreComponents } from '../../../core-components/package.json';
 import { version as corePluginApi } from '../../../core-plugin-api/package.json';
 import { version as e2eTestUtils } from '../../../e2e-test-utils/package.json';
-import { version as errors } from '../../../errors/package.json';
```

Comment:
There some things removed here that don't look to be added back?

Reply:
We don't seem to actually use the `@backstage/errors` package in `create-app` so this is fine 👍